src/main/java/org/beanpod/switchboard/controller/EncoderController.java

```diff
@@ -51,17 +47,19 @@
   private final MaintainDeviceStatus maintainDeviceStatus;
   private final HttpServletRequest request;
 
-  @GetMapping
-  public List<EncoderDto> retrieveAllEncoders() {
+  @Override
+  public ResponseEntity<List<EncoderModel>> retrieveAllEncoders() {
     UserEntity user = userDao.findUser(request.getUserPrincipal().getName());
 
     List<EncoderEntity> encoderEntities = encoderDao.getEncoders(user);
     maintainDeviceStatus.maintainStatusField(encoderEntities);
-    return encoderMapper.toEncoderDtos(encoderEntities);
+    List<EncoderDto> encoderDtos = encoderMapper.toEncoderDtos(encoderEntities);
+    List<EncoderModel> encoderModels = encoderMapper.toEncoderModels(encoderDtos);
+    return ResponseEntity.ok(encoderModels);
   }
```

Comment:
i noticed when retrieving the encoders the output object has an encoder object inside it which is repetitive 
![image](https://user-images.githubusercontent.com/41707836/114258378-1915ca00-9994-11eb-8d45-d43b19b651e8.png)
as opposed to master where it didnt have
![image](https://user-images.githubusercontent.com/41707836/114258406-4793a500-9994-11eb-8875-201c5e5c91ed.png)

was that what was meant by felix's comment?

Reply:
Yes, that is the same issue #477 will have a fix for that